# Edge infrastructure
uint16 time_stamp
uint16 processing_time
uint32 edge_id
uint16 edge_coordinate_system
float64 edge_x
float64 edge_y

# Objects
uint8 num_objects
Object[] objects